ambry-router/src/test/java/com/github/ambry/router/GetBlobOperationTest.java

```diff
@@ -186,55 +202,82 @@ public void after() {
   }
 
   /**
-   * Running for both {@link SimpleOperationTracker} and {@link AdaptiveOperationTracker} with and without encryption
-   * @return an array of Pairs of {{@link SimpleOperationTracker}, Non-Encrypted}, {{@link AdaptiveOperationTracker}, Encrypted}
-   * and {{@link AdaptiveOperationTracker}, Non-Encrypted}
+   * Running for both {@link SimpleOperationTracker} and {@link AdaptiveOperationTracker} with and without encryption,
+   * not include cloud backed data center.
+   * @return an array of {{@link SimpleOperationTracker}, Non-Encrypted, not-include-cloud},
+   * {{@link AdaptiveOperationTracker}, Encrypted, not-include-cloud}
+   * and {{@link AdaptiveOperationTracker}, Non-Encrypted, not-include-cloud}
    */
   @Parameterized.Parameters
   public static List<Object[]> data() {
-    return Arrays.asList(new Object[][]{{SimpleOperationTracker.class.getSimpleName(), false},
-        {AdaptiveOperationTracker.class.getSimpleName(), false},
-        {AdaptiveOperationTracker.class.getSimpleName(), true}});
+    return Arrays.asList(new Object[][]{
+        {SimpleOperationTracker.class.getSimpleName(), false, false},
+        {AdaptiveOperationTracker.class.getSimpleName(), false, false},
+        {AdaptiveOperationTracker.class.getSimpleName(), true, false}
+    });
   }
 
   /**
    * Instantiate a router, perform a put, close the router. The blob that was put will be saved in the MockServer,
    * and can be queried by the getBlob operations in the test.
    * @param operationTrackerType the type of {@link OperationTracker} to use.
    * @param testEncryption {@code true} if blobs need to be tested w/ encryption. {@code false} otherwise
+   * @param includeCloudDc {@code true} if add cloud backed DC to the cluster map. {@code false} otherwise
    */
-  public GetBlobOperationTest(String operationTrackerType, boolean testEncryption) throws Exception {
+  public GetBlobOperationTest(String operationTrackerType, boolean testEncryption, boolean includeCloudDc) throws Exception {
```

Comment:
I have a mixed feeling about adding cloud test in this file. I think this file is now too bloated to have more tests. Can we move these tests to a different file?

Reply:
Yes, I have the similar feeling. Will make it a standalone Cloud test.